java_training/src/com/lftechnology/december31/DateTest.java

```diff
@@ -0,0 +1,133 @@
+package com.lftechnology.december31;
+
+/**
+ * Write a test application named DateTest that demonstrates class Date's
+ * capabilities.
+ * 
+ * @author nerisa
+ * 
+ */
+public class DateTest {
+
+	public static void main(String[] args) {
+		DateUtility date = new DateUtility(2, 1, 2015);
```

Comment:
yah but this will print date like 2/0/2014 which is not valid ? Please check , better not to print invalid date